### PR TITLE
Building Mac OS X and window wheels

### PR DESCRIPTION
**Pull request by fgregg**
These scripts will 
- test pyhacrf on linux, mac os x, and windows
- on tagged commites, build and deploy wheels to pypi

@dirko, you'll need to 
1. enable travis ci for this repo: https://docs.travis-ci.com/user/getting-started/#To-get-started-with-Travis-CI%3A
2. enable appveyor for this repo: https://www.appveyor.com/docs

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+environment:
+  password:
+    secure: uOcv5TOm/CgwOmiLQhNX+g==
```
Comment (fgregg) [inline appveyor.yml:3]: If you want to appveyor to deploy the wheel to pypi, you'll need to create an encrypted variable for "your_pypi_password" and let me know what it is, I'll incorporate it into the script. https://www.appveyor.com/docs/build-configuration#secure-variables

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    - echo repository=https://pypi.python.org/pypi     >> ~/.pypirc
+    - echo username=zope.wheelbuilder                  >> ~/.pypirc
+    - echo password=$PYPIPASSWORD                      >> ~/.pypirc
+    - if [[ $TRAVIS_TAG && "$TRAVIS_OS_NAME" == "osx" ]]; then pip install twine; fi
+    - if [[ $TRAVIS_TAG && "$TRAVIS_OS_NAME" == "osx" ]]; then python setup.py bdist_wheel; fi
+    - if [[ $TRAVIS_TAG && "$TRAVIS_OS_NAME" == "osx" ]]; then twine upload dist/*; fi
+
+env:
+    global:
+        secure: J8y6Yv7w8z1pch0RzXquJwNG5ln78/rqo21TfnwEtymQnLO10jLw+YafwQozBlI8SWpU+k/C7UQCwtMAFEvBfgnToh+3fR70kX7qDvte9DKsbo2qUBAg/Pw/mVYHcpWVFW7wj9YVYxoe4Hk1vmmLuDxYu81/c75nxJljH5xdo+0=
```
Comment (fgregg) [inline .travis.yml:41]: If you want to deploy to pypi from travis, you'll need add your pypi password as an encrypted environmental variable. "PYPIPASSWORD=your_pypi_password"

https://docs.travis-ci.com/user/environment-variables/#Encrypted-Variables

**Comment by fgregg** [comment]
@dirko, can I help with anything else on this PR?

**Comment by dirko** [comment]
Great thanks! Will set up travis and appveyor a bit later today

**Comment by fgregg** [comment]
Great!

**Comment by fgregg** [comment]
ping?

**Comment by fgregg** [comment]
@dirko would you consider making me a collaborator on this project https://help.github.com/articles/adding-collaborators-to-a-personal-repository/

**Pull request closed without merging**